src/dragging/dragging.js

```diff
@@ -22,7 +22,22 @@
     let onMouseMove = function(e) {
       let deltaX = e.clientX - clientX + startX;
       let deltaY = e.clientY - clientY + startY;
-      onMove(deltaX, deltaY);
+      onMove(deltaX, deltaY, e);
+      var dragTargets = document.elementsFromPoint(e.clientX, e.clientY);
```

Comment:
Sorry, I misunderstood the code. I thought `dragTargets` implied that these are the things being dragged. Maybe `dropTargets` is a more appropriate name?